syncd/syncd.cpp

```diff
@@ -1313,7 +1313,7 @@ sai_status_t handle_generic(
 
                         sai_object_id_t switch_vid = redis_sai_switch_id_query(object_id);
 
-                        if (switches.at(switch_vid)->isDefaultCreatedRid(rid))
+                        if (switches.at(switch_vid)->isDiscoveredRid(rid))
```

Comment:
is this about vs switch? does not seem fit into this PR?

Reply:
this is because this change is on top of coldvids branch in other PR, since it will not compile without that